src/main/java/com/ays/assignment/model/mapper/AssignmentToAssignmentsResponseMapper.java

```diff
@@ -0,0 +1,35 @@
+package com.ays.assignment.model.mapper;
+
+import com.ays.assignment.model.Assignment;
+import com.ays.assignment.model.dto.response.AssignmentsResponse;
+import com.ays.common.model.mapper.BaseMapper;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+/**
+ * AssignmentEntityToAssignmentMapper is an interface that defines the mapping between an {@link Assignment} and an {@link AssignmentsResponse}.
+ * This interface uses the MapStruct annotation @Mapper to generate an implementation of this interface at compile-time.
+ * <p>The class provides a static method {@code initialize()} that returns an instance of the generated mapper implementation.
+ * <p>The interface extends the MapStruct interface {@link BaseMapper}, which defines basic mapping methods.
+ * The interface adds no additional mapping methods, but simply defines the types to be used in the mapping process.
+ */
+
+@Mapper
+public interface AssignmentToAssignmentsResponseMapper extends BaseMapper<Assignment, AssignmentsResponse> {
+
+    @Override
+    @Mapping(target = "location.longitude", source = "point.x")
+    @Mapping(target = "location.latitude", source = "point.y")
+    @Mapping(target = "status", source = "status")
```

Comment:
`status` alanı zaten doğrudan `status` alanına denk geldiği için özellikle belirtmemize ihtiyaç yok diye düşünüyorum

Reply:
İlk denediğimde status alanını maplemeyip null bıraktığı için böyle bir çözüme gittim.
Fakat şuan denediğimde mapliyor problem yok. Kaldırdım. @agitrubard 
Commit: `a5f60402517bb42ce73dff43641d0fa074a26922`